src/mixins/Projector.ts

```diff
@@ -215,14 +218,22 @@ export function ProjectorMixin<P, T extends Constructor<WidgetBase<P>>>(base: T)
 		}
 
 		public setChildren(children: DNode[]): void {
+			this._projectorChildren = [ ...children ];
```

Comment:
Ha nope of course I'm wrong, I do need to take the copy every time over wise my "frozen" properties/children will possibly be mutated then next time they are used to call `__setProperties__` or `__setChildren__`. Doh.